dev-docs/modules/kargo.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: bidder
+title: Kargo
+description: Prebid Kargo Bidder Adapter
+biddercode: Kargo
+gdpr_supported: true
+gvl_id: 972
+usp_supported: true
+coppa_supported: false
+schain_supported: false
+dchain_supported: false
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+floors_supported: true
+fpd_supported: true
```

Comment:
You're not allowed to claim you support FPD unless you explicitly document what is actually supported: site, user, per-imp, segments, etc.

Reply:
We plan on supporting `FPD` in the future but will change it to `false` for the time being.